test/scrabble_test.rb

```diff
@@ -4,9 +4,48 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+  def test_it_exists
+    game = Scrabble.new
+    assert_instance_of Scrabble, game
+  end
+
   def test_it_can_score_a_single_letter
-    skip
-    assert_equal 1, Scrabble.new.score("a")
-    assert_equal 4, Scrabble.new.score("f")
+    game = Scrabble.new
+    assert_equal 1, game.score("a")
+  end
+
+  def test_it_can_score_a_word
+    game = Scrabble.new
+    assert_equal 8, game.score("hello")
+  end
+
+  def test_case_name
+    game = Scrabble.new
+    assert_equal 0, game.score("")
+    assert_equal 0, game.score(nil)
+  end
+
+  def test_it_has_multipliers
```

Comment:
The tests below look great - I would suggest though only writing the next test once you have the previous method passing :)